docs/tutorials/rainbow-frontend-dev-tutorial.md

```diff
@@ -0,0 +1,308 @@
+# Ethereum & NEAR Rainbow Bridge Inoperability Tutorial for Devs
```

Comment:
add metadata/header to this doc as I added to the one above. :)

Reply:
added meta data :)